docs/reference/cli/resource/list.md

```diff
@@ -118,7 +118,7 @@ dsc resource list --tags Windows --tags Linux
 ```output
 Type                        Version  Requires  Description
 -------------------------------------------------------------------------------------------------------
-Microsoft.Windows/Registry  0.1.0              Registry configuration provider for the Windows Registry
+Microsoft.Windows/Registry  0.1.0              Registry configuration adapter for the Windows Registry
```

Comment:
As above, the description string should match the registry resource's description string:

```text
Manage Windows Registry keys and values
```